src/Semantics.js

```diff
@@ -96,6 +96,30 @@ Object.defineProperties(Wrapper.prototype, {
   // Returns the number of children of this CST node.
   numChildren: {get: function() { return this._node.numChildren(); }},
 
+  // Returns the minimum number of children for this CST node, if it's an iterator node.
+  // Otherwise, throws an exception.
+  minNumChildren: {
+    get: function() {
+      if (this.isIteration()) {
+        return this._node.minMatches;
+      }
+      throw new TypeError(
```

Comment:
I guess it depends on what you intend this would be used for. Depending on that, I could imagine that, e.g. an Apply node would return 0 and INFINITY for min and max, or just return min == num == max. It would mean you could just call these methods without having to do a type check first.  But if you don't think it makes, don't worry about it.

Can you actually give an example of a use case for these methods? Is it just another way to check if the node comes from a `*`, `+`, or `?`, or is it more generally useful?